### PR TITLE
config: add configs for publishing to maven central

### PR DESCRIPTION
**Pull request by kevindavee**

### CODE REVIEW

File: xendit-java-lib/build.gradle [hunk 1]
```diff
@@ -111,3 +123,76 @@ spotless {
         googleJavaFormat()
     }
 }
+
+publishing {
+    publications {
+        Production(MavenPublication) {
+            artifact("$buildDir/outputs/aar/xendit-java-release.aar") // replace with your library name
```
Comment (javiersuweijie) [inline xendit-java-lib/build.gradle:130]: should be `jar`

File: xendit-java-lib/build.gradle [hunk 2]
```diff
@@ -111,3 +123,76 @@ spotless {
...
+                authentication(userName: ossrhUser, password: ossrhPassword)
+            }
+
+            snapshotRepository(url: "https://s01.oss.sonatype.org/content/repositories/snapshots/") {
+                authentication(userName: ossrhUser, password: ossrhPassword)
+            }
+
+             pom.project {
+                name bintrayName
+                packaging 'aar'
```
Comment (javiersuweijie) [inline xendit-java-lib/build.gradle:170]: should be `jar`

**Review by javiersuweijie — CHANGES_REQUESTED** [review]
you might need to add this file to your root project so most of the fields are pre-filled (without the secrets and passwords)
https://github.com/xendit/xendit-sdk-android/blob/master/gradle.properties

File: xendit-java-lib/build.gradle (hunk 1 above)
Comment (kevindavee) [reply xendit-java-lib/build.gradle]: Changed accordingly @javiersuweijie

File: xendit-java-lib/build.gradle (hunk 2 above)
Comment (kevindavee) [reply xendit-java-lib/build.gradle]: Changed accordingly @javiersuweijie

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      name: Install gpg secret key
+      run: |
+        cat <(echo -e "${{ secrets.OSSRH_GPG_SECRET_KEY }}") | gpg --batch --import
+        gpg --list-secret-keys --keyid-format LONG
+    - id: publish-to-central
+      name: Publish to central repository
+      env:
+        MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+        MAVEN_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+      run: ./gradlew uploadArchives -Psigning.password=${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }}
```
Comment (javiersuweijie) [inline .github/workflows/publish.yml:34]: you need another step here to release after the upload.

`./gradlew closeAndReleaseRepository`

**Review by javiersuweijie — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 3 above)
Comment (kevindavee) [reply .github/workflows/publish.yml:34]: Updated accordingly @javiersuweijie

**Review by javiersuweijie — APPROVED** [review]
LGTM

**Comment by stanleynguyen** [comment]
Can we also update README and remove the step of adding bintray artifactory?

**Review by stanleynguyen — APPROVED** [review]

**Pull request merged**